ui/src/scenes/Editor/Monaco/questdb-sql/conf.ts

```diff
@@ -1,6 +1,7 @@
 import * as monaco from "monaco-editor"
 
 export const conf: monaco.languages.LanguageConfiguration = {
+  wordPattern: /(-?\d*\.\d\w*)|([^\`\~\!\@\#\$\%\^\&\*\(\)\-\=\+\[\{\]\}\\\|\;\:\"\,\<\>\/\?\s]+)/g,
```

Comment:
would be helpful to add a comment briefly explaining the purpose of this regex

Reply:
@argshook 
This is a derivative of default one (if not provided explicitly), like so:
https://github.com/microsoft/vscode/blob/main/src/vs/editor/common/core/wordHelper.ts#L31

The difference being we're allowing `'` and `.' to be included for the string to be considered a "word". 
In short, a default implementation won't consider `'table_name'` a word and thus won't highlight it if needed, whereas this change will make it work. The dot is also needed for table names created from CSV imports, for example.

Reply:
oh, i wasn't clear enough, i meant a comment in the code, for future generations :)

Reply:
@argshook Done 📝